specification/maps/data-plane/Route/stable/2025-01-01/route.json

```diff
@@ -0,0 +1,3783 @@
+        "type",
+        "geometry",
+        "properties"
+      ]
+    },
+    "InputWaypointProperties": {
+      "type": "object",
+      "description": "Specifies the properties of a Waypoint which is a specific location or point along a route or trip that serves as a reference or stopping point.",
+      "properties": {
+        "pointIndex": {
+          "description": "Identify and order the sequence of waypoints in the route. The default value is the `index value` of a features array.",
+          "format": "int64",
+          "type": "integer"
+        },
+        "pointType": {
+          "$ref": "#/definitions/InputWaypointTypeEnum"
+        }
+      }
+    },
+    "WaypointProperties": {
+      "x-ms-discriminator-value": "Waypoint",
+      "type": "object",
+      "description": "Specifies the properties of a Waypoint which is a specific location or point along a route or trip that serves as a reference or stopping point.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/FeatureProperties"
+        },
+        {
+          "$ref": "#/definitions/NavigationPointProperties"
+        }
+      ]
+    },
+    "ManeuverPointProperties": {
+      "x-ms-discriminator-value": "ManeuverPoint",
+      "type": "object",
+      "description": "Specifies the driving instructions and additional properties for each maneuver point in the route Leg.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/FeatureProperties"
+        },
+        {
+          "$ref": "#/definitions/NavigationPointProperties"
+        }
+      ]
+    },
+    "NavigationPointProperties": {
+      "description": "Waypoint and ManeuverPoint properties.",
+      "type": "object",
+      "properties": {
+        "routePathPoint": {
+          "$ref": "#/definitions/RoutePathPoint"
+        },
+        "order": {
+          "$ref": "#/definitions/Order"
+        },
+        "compassDirection": {
+          "description": "The direction of travel associated with a maneuver on a route, such as south or southwest. `Note`: Only supported for driving travelMode.",
+          "type": "string",
+          "readOnly": true
+        },
+        "steps": {
+          "description": "Steps between two consecutive maneuver points.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Step"
+          }
+        },
+        "instruction": {
+          "$ref": "#/definitions/Instruction"
+        },
+        "address": {
+          "description": "The address of the result. `Note`: Only supported for truck travelMode.",
+          "type": "object",
+          "properties": {
+            "countryRegion": {
+              "type": "object",
+              "properties": {
+                "ISO": {
+                  "description": "ISO of country/region represents an [ISO 3166-1 Alpha-2 region/country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)",
+                  "type": "string"
+                }
+              }
```

Comment:
The schema for `countryRegion` here is inconsistent with the `countryRegion` in `RegionTravelSummary`, which contains a single property `name`.